### PR TITLE
pd-ctl: return full region information for /region api

### LINKED ISSUE #1223: Return full region detail for /region API
Currently it only returns region meta, see: https://github.com/pingcap/pd/blob/f89771a4044c09b4f8cf67d7db8d72c4542556e3/server/api/region.go#L134-L138

Incomplete information can cause trouble when we want to find some regions. I think we can return full region information after done refactoring region clone.

/cc @nolouch

### PR DESCRIPTION
**Pull request by liukun4515**
### What problem does this PR solve? 

- return full region information not only the region meta.

close #1223 

### Check List 

Tests 

 - Unit test
 - Manual test

Side effects

no

Related changes

 - Need to update the documentation

### CODE REVIEW

**Comment by disksing** [comment]
Any need to add/update test cases?

**Comment by disksing** [comment]
The title should follow the rule mentioned in the development document https://github.com/pingcap/pd/blob/master/CONTRIBUTING.md#format-of-the-commit-message

**Comment by liukun4515** [comment]
No test case should be updated.
Just test by manual test through the simulator.
@disksing

File: server/api/region_test.go [hunk 1]
```diff
@@ -83,6 +83,32 @@ func (s *testRegionSuite) TestRegion(c *C) {
 	c.Assert(r2, DeepEquals, newRegionInfo(r))
 }
 
+func (s *testRegionSuite) TestRegions(c *C) {
+	rs := []*core.RegionInfo{
+		newTestRegionInfo(2, 1, []byte("a"), []byte("b")),
+		newTestRegionInfo(3, 1, []byte("b"), []byte("c")),
+		newTestRegionInfo(4, 2, []byte("c"), []byte("d")),
+	}
+	regions := make([]*regionInfo, 0)
```
Comment (disksing) [inline server/api/region_test.go]: make([]*regionInfo, 0, len(rs))

**Comment by disksing** [comment]
PTAL @nolouch @rleungx

**Pull request merged**